articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Azure AD v2 JavaScript SPA quickstart | Microsoft Docs
+description: Learn how JavaScript SPA applications can call an API that require access tokens by Azure Active Directory v2.0 endpoint
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mtillman
+editor: ''
+
+ms.service: active-directory
+ms.component: develop
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity
+ms.date: 07/10/2018
+ms.author: andret
+ms.custom: aaddev
+
+#Customer intent: As an app developer, I want to learn how to get access tokens using the Azure AD v2.0 endpoint so that my JavaScript app can sign in personal and work and school accounts.
+---
+
+# Sign in users and acquire an access token from a JavaScript application
+
+In this quickstart, you'll learn how to use a code sample that demonstrates how a JavaScript single page application (SPA) can sign in personal accounts, work and school accounts, get an access token, and call the Microsoft Graph API.
+
+![How the sample app generated by this Quickstart works](media/active-directory-javascriptspa/javascriptspa-intro.png)
+
+> [!div renderon="docs"]
+> ## Register your application and download your quickstart app
+> You have two options to start your quickstart application:
+> * [Express] [Option 1: Register and then download your quickstart application](#option-1-register-your-application-and-download-your-quickstart-app)
```

Comment:
@andretms I would clarify this as Register and then download the configured sample. Option 2: is register and configure the sample.